code/modules/admin/verbs/randomverbs.dm

```diff
@@ -47,6 +47,52 @@
 	admin_ticket_log(M, msg)
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Subtle Message") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_mod_antag_rep(client/C in GLOB.clients, var/operation)
+	set category = "Special Verbs"
+	set name = "Modify Antagonist Reputation"
+
+	if (!holder)
+		to_chat(src, "Only administrators may use this command.")
+		return
+
+	var/msg = ""
+	var/log_text = ""
+
+	if(operation == "zero")
+		log_text = "Set to 0"
+		SSpersistence.antag_rep.Remove(C.ckey)

```

Comment:
-=

Reply:
I'm not sure what you mean here. This works as intended. The four admin options are [increase] [decrease] [set] [zero], where [zero] simply removes all tickets (removes them from the data file).

Reply:
No i mean, `SSpersistence.antag_rep -= C.ckey`